src/util.js

```diff
@@ -410,6 +410,52 @@
       } catch(error) {
         return false;
       }
+    },
+
+    /**
+     * Decide if data should be treated as binary based on the content
+     * and content-type.
+     *
+     * @param {string} content - The data
+     * @param {string} mimeType - The data's content-type
+     *
+     * @returns {boolean}
+     */
+    shouldBeTreatedAsBinary (content, mimeType) {
+      return !mimeType || mimeType.match(/charset=binary/) || /[\x00-\x1F]/.test(content);
```

Comment:
I don't understand the first condition here. Why is it considered binary when the mimeType is unknown? Wouldn't it be more likely that it's text with RS documents, but have to be looked into especially in that case?

Reply:
Yes, I'm not sure about this either. To be honest, I just used that because it already was like that before. I just added the last condition.

Reply:
Especially with that new condition, the first condition should probably be removed.

Reply:
Yup.